table_helper.cc

```diff
@@ -95,6 +109,14 @@ future<> table_helper::insert(service::query_state& qs, noncopyable_function<cql
     }).discard_result();
 }
 
+future<> table_helper::insert(service::query_state& qs, noncopyable_function<cql3::query_options (bool)> opt_maker) {
+    return cache_table_info(qs).then([this, &qs, opt_maker = std::move(opt_maker)] () mutable {
```

Comment:
Why is mutable needed here?

Reply:
Looks like I repeated the mistake from the other table_helper::insert overload